### ISSUE TITLE
Linking / Routing not working

### ISSUE THREAD

**Issue by reblevins** [inkline/inkline#366]
* **What is the expected and current behavior?**
  NOTE: I first looked at #336 but nothing suggested there seemed to help.

  When using `INavItem` and either `to="/leads"` or `:to="{ name: 'leads' }"`, it does not work as expected.

* **Steps to reproduce**
```
<INavbar class="_padding-x:6! _padding-y:2! _background:primary!">
  <INavbarBrand to="/">
    <img
      class="logo q-mr-md"
      src="..."
    >
  </INavbarBrand>
  <INavbarCollapsible class="_display:flex! _justify-content:space-between!">
    <INav>
      <INavItem :to="{ name: 'leads' }" class="_margin-right:3">
        Leads
      </INavItem>
    </INav>
    <!-- other nav items -->
  </INavbarCollapsible>
</INavbar>
```

* **Please tell us about your environment:**
    - Version: >=4.0.0
    - Browser: Have only tested in Chrome 114.0.5735.106 (Official Build) (arm64) | MacOS Safari Version 16.5 (18615.2.9.11.4)
    - Language: ES

* **Other information**
  This is the HTML it produces:
```
<div class="_margin-right:3 nav-item" to="[object Object]" tabindex="0" tag="div" data-v-01675290="">
  <!-- @slot default Slot for default nav item content -->
  Leads
</div>
```
  
  Router Link works as expected:
```
<router-link :to="{ name: 'leads' }" class="_margin-right:3">
  Leads
</router-link>
```

**Comment by alexgrozav** [comment]
Just to make sure, this is a standard `vite` setup, not a `nuxt` setup, right?

**Comment by alexgrozav** [comment]
Should be fixed in today's release: `4.0.2`. Can you please try and confirm?
